doc/source/contributing_code.rst

```diff
@@ -69,32 +69,38 @@ You can do this either with :ref:`conda <dev-with-conda>` or :ref:`pip <dev-with
 
 .. _dev-with-conda:
 
-With conda
-----------
+With conda-forge packages
+-------------------------
+
+We will use pre-compiled packages from the `conda-forge <https://conda-forge.org/>`_ repository.
+
+The program to manage these packages is called :program:`conda` although for the following we recommend using the faster one called :program:`mamba`. We will use ``mamba`` for the examples but you can equally use ``conda`` instead.
```

Comment:
I think if one uses `conda` instead of `mamba`, the default channel is not `Conda-forge`. So if the title is "conda-forge" packages, we need to word this a bit more carefully. We could mention
```
conda config --add channels conda-forge
```
but maybe it just makes things a bit more confusing.

Reply:
All our installation instructions below explicitly use `-c conda-forge` and we do not rely on default channels. Do you still think this needs to be made more clear?

Reply:
My bad, I missed that on the commands.